blog-article.html

```diff
@@ -0,0 +1,317 @@
+            <a href="./catalog.html">COLLECTION</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./faqs.html">FAQS</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="#" class="dropdown">PAGES</a>
+            <ul class="dropdown-content">
+              <li>
+                <a href="./catalog.html">NFT Catalog</a>
+              </li>
+              <li>
+                <a href="./blog.html">Blog</a>
+              </li>
+              <li>
+                <a href="./contact-us.html">Contact Us</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="btn-box">
+        <button class="switch-mode">
+          <img src="./img/btn-icons/switch.png" alt="" />
+        </button>
+
+        <a href="https://discord.com/" class="header-link" target="_blank">
+          <img src="./img/socialntw-icons/discord-btn.png" alt="" />
+          DISCORD
+        </a>
+
+        <button class="header-btn">
+          <img src="./img/btn-icons/wallet.png" alt="" />
+          CONNECT
+        </button>
+      </nav>
+    </header>
+
+    <!-- Main -->
+    <main>
+      <!-- Breadcrumbs -->
+      <section class="breadcrumbs">
+        <h1>BLOG DETAIL</h1>
+        <ul>
+          <li><a href="./index.html">HOME</a></li>
+          <li><a href="./blog.html">OUR BLOG</a></li>
+          <li><a href="#blog-detail">BLOG DETAIL</a></li>
+        </ul>
+      </section>
+
+      <section class="blog-article" id="blog-detail">
+        <img
+          src="./img/img-blog/img-blog-article_main.png"
+          alt="The man standing infront of the giant green dragon"
+        />
+        <h2>Enior Apple Employee Alleges Sexism At</h2>
+        <p>GAMING</p>
+        <ul>
+          <li>
+            <img src="./img/btn-icons/person-icon.png" alt="" />
+            Tony Nguyen
+          </li>
+          <li class="blog-date">
+            <img src="./img/btn-icons/calendar-icon.png" alt="" />
+            20 Jun 2022
+          </li>
+        </ul>
+
+        <p class="blog-story">
+          The Basilisks that players collect are represented with NFTs on
+          Ethereum’s blockchain with real-world value. Players use the Basilisks
+          they have gathered to battle other players to win ether (ETH). In
+          Balthazar Dragons, players are immersed in a 3D open world to explore
+          and capture dragon-like beasts called Basilisks. Balthazar Dragons is
+          an upcoming fantasy role-playing game developed on the Ethereum
+          blockchain by a decentralized autonomous organization (DAO) called the
+          Balthazar DAO.
+        </p>
+
+        <blockquote>
+          “The first huge metaverse arena to arrange live sports, hang out and
+          even make bets and wagers”
+          <ul>
+            <li class="blog-article-rating">&starf; 8.5/ 10</li>
+            <li>Markout Corporation</li>
```

Comment:
for quote resourse you can use tag `cite` 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite